src/Components/Register/Register.js

```diff
@@ -21,59 +21,155 @@ const Register = ({ onSubmit, errorLabel }) => {
     password: '',
     repeatPassword: '',
     alterEgo: '',
-    icon: '' });
+    icon: '',
+    isPwdMatched: false
+  });
 
   const handleChange = ({ target }) => {
-    const { name, value } = target;
+
+  const { name, value } = target;
+
+  let checkPwdValidity = {};
+    if ( name === 'password' ) {
+        if ( value === payload.repeatPassword && value !== '' ) {
+          checkPwdValidity.isPwdMatched = true;
+        } else {
+          checkPwdValidity.isPwdMatched = false;
+        };
+      };
+    if ( name === 'repeatPassword' ) {
+        if ( value === payload.password && value !== '' ) {
+          checkPwdValidity.isPwdMatched = true;
+        } else {
+          checkPwdValidity.isPwdMatched = false;
+        };
+      };
     setPayload({
       ...payload,
+      ...checkPwdValidity,
       [name]: value
     });
   };
 
   const handleSubmit = e => {
     e.preventDefault();
-    onSubmit(payload);
+    if ( payload.isPwdMatched ) {
+      onSubmit( payload );
+    } else {
+      alert( "Check password" )
```

Comment:
Please add semicolon at the end of line.

Reply:
Done.